src/devices/yandex.ts

```diff
@@ -126,32 +127,63 @@ function YandexCluster(): ModernExtend {
     });
 }
 
-const reinterview: OnEvent = async (type, data, device, settings, state) => {
-    if (type == 'deviceAnnounce') {
-        // reinterview
-        try {
-            await device.interview(true);
-            logger.info(`Succesfully interviewed '${device.ieeeAddr}'`, NS);
-            // bind extended endpoint to coordinator
-            for (const endpoint of device.endpoints) {
-                if (endpoint.supportsOutputCluster('genOnOff')) {
-                    await endpoint.bind('genOnOff', 1);
+function reinterview(): ModernExtend {
+    let coordEnd: Endpoint | number = 1;
+    let localMeta: Fz.Meta;
+    const configure: Configure[] = [
+        async (device, coordinatorEndpoint, definition) => {
+            coordEnd = coordinatorEndpoint;
+        },
+    ];
+    const fromZigbee: Fz.Converter[] = [
+        {
+            cluster: 'genOnOff',
+            type: ['attributeReport', 'readResponse'],
+            convert: (model, msg, publish, options, meta) => {
+                localMeta = meta;
+                return;
+            },
+        } satisfies Fz.Converter,
+        {
+            cluster: 'genBasic',
+            type: ['attributeReport', 'readResponse'],
+            convert: (model, msg, publish, options, meta) => {
+                localMeta = meta;
+                return;
+            },
+        } satisfies Fz.Converter,
+    ];
+    const onEvent: OnEvent = async (type, data, device, settings, state) => {
```

Comment:
I think it would be better to pass `deviceExposesChanged` as an argument of event, preferably add `meta` there which is an object containing `deviceExposesChanged`. Because currently the code will not work when no genBasic/genOnOff message has been received yet.